packages/uniforms/__tests__/AutoForm.tsx

```diff
@@ -27,66 +31,84 @@ describe('AutoForm', () => {
     validator.mockClear();
   });
 
-  describe('when changed', () => {
+  describe('when changes', () => {
```

Comment:
I think the previous naming convention was better. If you have any arguments for changing it, let's discuss it.

Reply:
fixed